openmc/mgxs/mgxs.py

```diff
@@ -149,6 +149,8 @@ class MGXS(object):
         Name of the multi-group cross section
     rxn_type : str
         Reaction type (e.g., 'total', 'nu-fission', etc.)
+    nu : bool
```

Comment:
I'm not sure I'm a fan of adding `nu` as a property of for *all* `MGXS` subclasses. What is your reasoning? I'll think about it, but lean towards making it a property of only the `ScatterXS`, `ScatterMatrixXS` and `FissionXS` classes.